packages/x-data-grid-generator/package.json

```diff
@@ -50,8 +50,8 @@
   "peerDependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
-    "@mui/icons-material": "^5.4.1 || ^6.0.0 || ^7.0.0 || ^7.0.0-beta",
-    "@mui/material": "^5.15.14 || ^6.0.0 || ^7.0.0 || ^7.0.0-beta",
+    "@mui/icons-material": "^7.0.0",
```

Comment:
@Janpot could this package peer dep version be kept with looser range?
Or will anything before v7 also cause ESM issues? 🤔

Reply:
v6 should be safe as this package had the exports field already. but I don't see why you would upgrade `@mui/material` and not icons, it's no work.

Reply:
🤔 not every v6 though

Reply:
Nevermind, since we agreed to keep support for previous majors for now.